docs/pages/guides/emojimon/2-getting-started.mdx

```diff
@@ -15,26 +15,26 @@ Most projects in MUD are started by running `pnpm create mud <project name>`, bu
    You can use the command below.
 
    ```bash copy
-   git clone https://github.com/latticexyz/emojimon.git -b mud-update-next-14
+   git clone https://github.com/latticexyz/emojimon.git -b mud-update-next-16
    ```
 
 1. Install the dependencies and start up the MUD services (anvil node, contracts deployer, and client).
 
    ```bash copy
-   cd emojimon && pnpm install && pnpm run dev
+   cd emojimon && pnpm install && pnpm dev
    ```
 
 1. At this point you can play the game to see the finished game.
    Open the game in your browser.
-   By default the game is at [http://localhost:3000](http://localhost:3000), but it could be on a different port if that port's taken.
+   By default the game is at [http://localhost:3000](http://localhost:3000).
 
 1. Click the map to spawn at a specific location and move using the arrow keys.
    Note that on each block you can only move one space, and the block time on the anvil blockchain is one second.
 
-1. Now that you've seen our end goal, reset the repository to [the starting snapshot](https://github.com/latticexyz/emojimon/tree/8520a8206929f6265609cc9166c985846fd0cd87).
+1. Now that you've seen our end goal, reset the repository to [the starting snapshot](https://github.com/latticexyz/emojimon/tree/2e1bde1f1462a1838cd9091e2dbd9f531a75bbb5).
 
 ```sh copy
-git reset --hard 8520a8206929f6265609cc9166c985846fd0cd87
+git reset --hard 9dd3616244bc25518b06dfaa7beda51da07fb750
```

Comment:
this commit is different from the one linked above?

Reply:
Oops, the one in the command is the correct one. I'll fix it.
The main difference is viem is upgraded to a version that actually works with our latest.